### PR TITLE
Added handling hidden parameter in commands.

### PR DESCRIPTION
**Pull request by Gargy007**
Issue: #8

Just check that code if it's OK for you. Thank you

### CODE REVIEW

File: click_command_tree.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ def _build_command_tree(click_command):
 
     if isinstance(click_command, click.core.Group):
         for _, cmd in click_command.commands.items():
-            wrapper.children.append(_build_command_tree(cmd))
+            if not cmd.hidden:
```
Comment (whwright) [inline click_command_tree.py:33]: It looks like the `hidden` attribute was added in v7, and I would like to keep this library as backwards compatible as possible. Could this also handle the case where this attribute is not there?

```suggestion
            if not getattr(cmd, "hidden", False): 
```

**Review by whwright — COMMENTED** [review]
Thanks for the contribution!

File: click_command_tree.py (hunk 1 above)
Comment (Gargy007) [reply click_command_tree.py]: Sure,
 the offered solution was just proof of concept. Thank you for reply

**Comment by Gargy007** [comment]
Hi, 
 the last one question :-) Do you have any plan / schedule to update the click-tree with this hidden commands and get it on pypi.org ? 
 Thanks :-)

**Comment by whwright** [comment]
> Hi, the last one question :-) Do you have any plan / schedule to update the click-tree with this hidden commands and get it on pypi.org ? Thanks :-)

I will make some time this weekend to get this release out

**Comment by whwright** [comment]
@Gargy007 I released version 1.1.1

**Comment by Gargy007** [comment]
Hi,
 thank you, I appreciate it.

Petr

pá 3. 3. 2023 v 15:07 odesílatel Harrison Wright ***@***.***>
napsal:

**Pull request merged**